Adding Library for step 7

marktab opened the pull request:
Need to add  
# from pyspark.sql import *

on step 7 before the Row() command (sorry I don't seem to be able to make the edits using the web app properly)

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

marktab committed:
Adding Library for step 7

Need to add  
from pyspark.sql import *

on step 7 before the Row() command (sorry I don't seem to be able to make the edits using the web app properly)
